drivers/spi/spi_silabs_eusart.c

```diff
@@ -236,45 +749,49 @@ static int spi_silabs_eusart_init(const struct device *dev)
 	if (err < 0) {
 		return err;
 	}
-	spi_context_unlock_unconditionally(&data->ctx);
 
-	return 0;
-}
+	spi_context_unlock_unconditionally(&data->ctx);
 
-static int spi_silabs_eusart_transceive(const struct device *dev, const struct spi_config *config,
-				       const struct spi_buf_set *tx_bufs,
-				       const struct spi_buf_set *rx_bufs)
-{
-	struct spi_silabs_eusart_data *data = dev->data;
-	uint16_t control = 0;
-	int ret;
+	pm_device_init_off(dev);
 
-	spi_context_lock(&data->ctx, false, NULL, NULL, config);
+	err = pm_device_runtime_enable(dev);
+	
+	pm_device_driver_init(dev, spi_silabs_eusart_power_management_action_pm_action);
```

Comment:
The return value of `pm_device_driver_init()` must be returned from the init function.

Reply:
Also typo: s/power_management_action_pm_action/pm_action/